src/pdm/cli/commands/run.py

```diff
@@ -341,11 +341,8 @@ def run_task(
         elif kind == "composite":
             assert isinstance(value, list)
 
-        self.project.core.ui.echo(
-            f"Running {task}: [success]{args}[/]\n",
-            err=True,
-            verbosity=termui.Verbosity.NORMAL,
-        )
+        self.display_task(task, args)
```

Comment:
There is one minor thing, when task type is `composite`, the `args` list is just empty, can you change it to show something meaningful like sub commands?

Reply:
PR updated with additional test cases for composite 👍🏼 
I reused the `Task.short_description` property already used in `pdm run -l` so it uses the same rules and formatting.